snapshot_deploy.sh

```diff
@@ -2,8 +2,8 @@
 
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 export GPG_KEY_ID="Snowflake Computing"
-export LDAP_USER="$sonatype_user"
-export LDAP_PWD="$sonatype_password"
+export LDAP_USER="$INTERNAL_NEXUS_USERNAME"
```

Comment:
could you update unshaded_snapshot_deploy.sh as well?

Reply:
Sure will update it.